### ISSUE TITLE
Embedded video not playing in baker

### ISSUE THREAD

**Issue by Appmantx** [Simbul/baker#1021]
Hello, This is my first app,so I am not sure. In xcode, I have embeded video from my s3 accout,I also I an interactive forms. I set set the simulator and a seperate testing account with Itunes. I downloads fine, view pages fine, The interactive forms work but the video's and hyperllinks to other sites do not work. It this OK? Should everythink work in test mode?

**Comment by folletto** [comment]
Yes. Everything must work (with the exceptions documented on Apple's side, like push notifications).

**Comment by Appmantx** [comment]
Ok, I got the hyperlinks to work but can't get the video to play
Please see video https://www.youtube.com/watch?v=DCrY8t-Ri3U

**Comment by Appmantx** [comment]
Is there anyone using Video in there baker magazine?

**Comment by tekdope** [comment]
On S3, did you set the video file as public?

Envoyé de mon iPad

**Comment by Appmantx** [comment]
yes, html page works fine, permissions are okay

**Comment by chris-82** [comment]
Hi,

how did you embed the video to your HTML-document? I'm using this one and it works fine:

< video controls poster="http://my-url.nic/videos/138291383.png" width="500px" height="295px">
< source src="http://my-url.nic/videos/138291383.mp4" type='video/mp4; codecs="avc1.42E01E, mp4a.40.2"' />
</ video >

**Comment by Appmantx** [comment]
I used Indesign with the in5 extention. Exoports fine, I I go to the the direct url, it works but the video doesn't play in baker

**Comment by Appmantx** [comment]
I am going back and try this with hype

**Comment by folletto** [comment]
Please check also the thread #927.

**Comment by justinputney** [comment]
If you're exporting from in5, it may be worth updating to version 1.5.4 or higher.

It looks like mediaelementjs and the Baker container do not play well together. As of 1.5.4, the in5 code detects the Baker container by checking the userAgent for "BakerFramework". If detected, in5 bypasses the mediaelementjs initialization and uses the native audio/video elements.
